mesa_geo/visualization/modules/MapVisualization.py

```diff
@@ -1,13 +1,38 @@
+import dataclasses
+from dataclasses import dataclass
+
 import geopandas as gpd
 from folium.utilities import image_to_url
 from mesa.visualization.ModularVisualization import VisualizationElement
-from shapely.geometry import mapping
+from shapely.geometry import mapping, Point
 
 from mesa_geo.raster_layers import RasterLayer, RasterBase
 
 
+@dataclass
+class LeafletPortrayal:
+    """A dataclass defining the portrayal of a GeoAgent in Leaflet map.
+
+    The fields are defined to be consistent with GeoJSON options in
+    Leaflet.js: https://leafletjs.com/reference.html#geojson
+    """
+
+    style: dict = None
```

Comment:
Should this be `Dict[str, Any] | None`? Or do the values have a known type?

Reply:
It seems to be `Dict[str, str | bool | int | float] | None`, but `Dict[str, Any] | None` is neater and more flexible. I'll change to this.

Reply:
Changed to `Dict[str, Any] | None`.